resources/views/service/markets/edit.blade.php

```diff
@@ -0,0 +1,50 @@
+@extends('service.layouts.app')
+@section('content')
+
+    <div id="container">
+        @include('service.includes.sidebar')
+        <div id="main-content">
+
+            <h1>Edit Market</h1>
+
+            <p>Edit a Market</p>
```

Comment:
this might be redundant

Reply:
yeah; I'm thinking sopmething more... explainy could go there.